pkg/devfile/parser/context/context.go

```diff
@@ -75,6 +75,18 @@ func (d *DevfileCtx) Populate() (err error) {
 	return d.populateDevfile()
 }
 
+// SetAbsPath sets absolute file path for devfile
+func (d *DevfileCtx) SetAbsPath() (err error) {
+	// Get devfile absolute path
+	if d.absPath, err = util.GetAbsPath(d.relPath); err != nil {
+		return err
+	}
+	klog.V(4).Infof("absolute devfile path: '%s'", d.absPath)
```

Comment:
this should be `2` according to https://github.com/openshift/odo/pull/3876/files

Reply:
done.